src/option_parser.c

```diff
@@ -109,6 +110,19 @@ char *get_value(char *section, char *key)
         return NULL;
 }
 
+char *ini_get_path(char *section, char *key, const char *def)
+{
+        char *path = ini_get_string(section, key, def);
+
+        wordexp_t replaced_path;
+        if (path && 0 == wordexp(path, &replaced_path, WRDE_NOCMD)){
+                path = g_strdup(replaced_path.we_wordv[0]);
```

Comment:
You're also only getting the first word here, we_wordv is a string array (`char **`), not a `char *`. It's a common mistake so please also add a regression test for this.